# MotorSpdTar.msg

uint16 num
float64[] spd_tar
float64[] time_dur